qutebrowser/mainwindow/prompt.py

```diff
@@ -661,8 +675,11 @@ def _set_fileview_root(self, path, *, tabbed=False):
             log.prompt.exception("Failed to get directory information")
             return
 
-        root = self._file_model.setRootPath(path)
-        self._file_view.setRootIndex(root)
+        self._root_index = self._file_model.setRootPath(path)
+        self._file_view.setRootIndex(self._root_index)
+
+        self._directories_hide_show_model(path)
+        self._file_view.setModel(self._file_model)
```

Comment:
Why is a `setModel` needed here?

Reply:
That was from when I used a QSortFilterProxyModel and needed to use it. Will remove.